buildsystem/codecompliance/authors.py

```diff
@@ -6,6 +7,45 @@
 
 import re
 
+def get_email_obfuscation_substitutions():
+    """
+    What tokens in email should be replaced with
+    """
+    return {
+    '.' : ' noot ',
```

Comment:
I wanted to avoid `dot`, because it's an obvious thing for a bot to try. [`noot`](https://www.youtube.com/watch?v=a4VvRWTD3Ok) is nonsense, so I thought it might give us some protection. but `dawt` is probably good enough, and more guessable.